src/it/resources/data/typicalDataBundle.json

```diff
@@ -271,6 +326,24 @@
       "isClosingEmailEnabled": true,
       "isPublishedEmailEnabled": true,
       "isPublishedEmailSent": false
+    },
+    "session1InCourse2": {
+      "id": "00000000-0000-4000-8000-000000000704",
+      "course": {
+        "id": "idOfCourse2"
+      },
+      "name": "First feedback session",
+      "creatorEmail": "instr1@course2.tmt",
+      "instructions": "Please please fill in the following questions.",
+      "startTime": "2012-04-01T22:00:00Z",
+      "endTime": "2027-04-30T22:00:00Z",
+      "sessionVisibleFromTime": "2012-03-28T22:00:00Z",
```

Comment:
Why do we need a new session? What's makes this session unique?

Reply:
Have removed this. Wanted to have a session and question with no feedback responses at all. Now using existing data 👍🏻